GH-131278: Support building using "computed gotos" for clang-cl on Windows (#131279)
python/cpython: Misc/NEWS.d/next/Build/2025-03-15-12-32-56.gh-issue-131278.1nd0mJ.rst

Add optimizing flag ``WITH_COMPUTED_GOTOS`` to support such builds using

> clang-cl on Windows. Patch by Chris Eibl.

Reviewer: ```suggestion
:prog:`clang-cl` on Windows. Patch by Chris Eibl.
```

The `prog` role is a bit unknown but it may help in the future if we change how it's being rendered.

> Add optimizing flag ``WITH_COMPUTED_GOTOS`` to support such builds using
> clang-cl on Windows. Patch by Chris Eibl.

Reviewer 2: ```suggestion
Add optimizing flag ``WITH_COMPUTED_GOTOS`` to Windows builds for when
using a compiler that supports it (currently clang-cl). Patch by Chris Eibl.
```

> :prog:`clang-cl` on Windows. Patch by Chris Eibl.

Reviewer: Arf, the `prog` role seems to fail, it was `program` (https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-program) Sorry :(

Author: No worries, I'll revert ...
